test/old-api/env.js

```diff
@@ -647,63 +601,65 @@ describe("jsdom/env", () => {
       });
     });
 
-    specify(
-      "with configurable resource loader modifying routes and content",
-      { async: true },
-      t => {
-        const routes = {
-          "/js/dir/test.js": "window.modifiedRoute = true;",
-          "/html": "<!DOCTYPE html><html><head><script src='./test.js'></script></head><body></body></html>"
-        };
-
-        const server = http.createServer((req, res) => {
-          res.writeHead(200, { "Content-Length": routes[req.url].length });
-          res.end(routes[req.url]);
-        });
-
-        const time = new Date(Date.now() + 24 * 60 * 60 * 1000);
-        const cookie = "key=value; expires=" + time.toGMTString() + "; path=/";
-
-        server.listen(64001, "127.0.0.1", () => {
-          env({
-            document: { cookie },
-            url: "http://127.0.0.1:64001/html",
-            resourceLoader(resource, callback) {
-              assert.ok(typeof resource === "object");
-              assert.ok(typeof resource.url === "object");
-              assert.equal(resource.cookie, "key=value");
-              assert.equal(resource.baseUrl, "http://127.0.0.1:64001/html");
-              assert.ok(typeof resource.defaultFetch === "function");
-              assert.ok(typeof callback === "function");
-              if (/\.js$/.test(resource.url.path)) {
-                resource.url.path = "/js/dir" + resource.url.path;
-                resource.defaultFetch((err, body) => {
-                  if (err) {
-                    callback(err);
-                  } else {
-                    callback(null, body + "\nwindow.modifiedContent = true;");
-                  }
-                });
-              } else {
-                resource.defaultFetch(callback);
-              }
-            },
-            done(err, window) {
-              server.close();
-              assert.ifError(err);
-              assert.ok(window.modifiedRoute);
-              assert.ok(window.modifiedContent);
-              t.done();
-            },
-            features: {
-              FetchExternalResources: ["script"],
-              ProcessExternalResources: ["script"],
-              SkipExternalResources: false
-            }
-          });
-        });
-      }
-    );
+    // TODO: With the current implementation, this is not possible.
+    // QUESTION: Is this important to have in the new resource loader?
```

Comment:
It's possible by overriding fetch, right?

Reply:
It wasn't, but I have done some changes to make it possible.